<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>728</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>661</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="accountsWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="categoriesWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="totalsAccounts">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>211</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="totalsCategories">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>256</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSatie_Finances">
    <property name="title">
     <string>Finances</string>
    </property>
    <addaction name="actionSetDir"/>
    <addaction name="separator"/>
    <addaction name="actionLoadYearMonth"/>
    <addaction name="actionNextMonth"/>
    <addaction name="actionPreviousMonth"/>
    <addaction name="actionLoadMonth"/>
    <addaction name="separator"/>
    <addaction name="actionGenerateNextMonth"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuThis_Month">
    <property name="title">
     <string>This Month</string>
    </property>
    <addaction name="actionNewAccount"/>
    <addaction name="actionNewIncome"/>
    <addaction name="actionNewCategory"/>
    <addaction name="separator"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionEditFileUnderCursor"/>
    <addaction name="separator"/>
    <addaction name="actionReloadFromFiles"/>
   </widget>
   <widget class="QMenu" name="menuThis_File">
    <property name="title">
     <string>This File</string>
    </property>
    <addaction name="actionIsIncome"/>
    <addaction name="actionIsAccount"/>
    <addaction name="actionIsBusiness"/>
    <addaction name="actionSetBudget"/>
    <addaction name="separator"/>
    <addaction name="actionNewActivity"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFile"/>
   </widget>
   <addaction name="menuSatie_Finances"/>
   <addaction name="menuThis_Month"/>
   <addaction name="menuThis_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetDir">
   <property name="text">
    <string>Set Working Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLoadYearMonth">
   <property name="text">
    <string>Load Year-Month</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionLoadMonth">
   <property name="text">
    <string>Load Month</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewAccount">
   <property name="text">
    <string>New Account File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGenerateNextMonth">
   <property name="text">
    <string>Generate Next Month</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionNewCategory">
   <property name="text">
    <string>New Save/Spend Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNewIncome">
   <property name="text">
    <string>New Income File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionNewActivity">
   <property name="text">
    <string>New Activity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionIsIncome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Is Income</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSetBudget">
   <property name="text">
    <string>Set Budget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionIsAccount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Is Account</string>
   </property>
  </action>
  <action name="actionEditFileUnderCursor">
   <property name="text">
    <string>Edit File Under Cursor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionNextMonth">
   <property name="text">
    <string>Next Month</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPreviousMonth">
   <property name="text">
    <string>Previous Month</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionIsBusiness">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Is Business</string>
   </property>
  </action>
  <action name="actionReloadFromFiles">
   <property name="text">
    <string>Reload From Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSetDir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>requestDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>requestDirectory()</slot>
 </slots>
</ui>
